articles/openshift/oshift-how-v4-update.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Supported customer update strategy for version 4 clusters
+description: Provides information on supported strategies for self-service customer updates
+services: openshift
+author: Kieran O'Neill
+reviewer: 
+lastreviewed: 24/07/2020
+toc_rootlink: Reference
+toc_sub1: v4
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Supported customer update strategy for version 4 clusters
+toc_fullpath: Reference/oshift-how-v4-update.md
+toc_mdlink: oshift-how-v4-update.md
+---
+
+# Supported customer update strategy for version 4 clusters
+
+## Update Information
+
+In OpenShift version 4, updates of the cluster and operating system of the nodes have been exposed to the customer through the cluster. You can update clusters through the UI and the cluster will then deal with updating the relevant operator components and the OS version of the RHCOS nodes running as masters, infras and workers. The OS version of the nodes is tied to the version of the cluster and is thoroughly tested by Red Hat pipelines prior to release.
```

Comment:
Need to update master terminology as per other OpenShift doc

Reply:
Changed master to controlplane in latest commit